src/core_modules/capture-core/components/TrackedEntityTypeSelector/TrackedEntityTypeSelector.component.js

```diff
@@ -51,7 +51,11 @@ export const TrackedEntityTypeSelectorPlain =
                       label={headerText}
                       onChange={handleSelectionChange}
                       selected={selectedSearchScopeId}
-                      empty={<div className={classes.customEmpty}>Custom empty component</div>}
+                      empty={
+                          <div className={classes.customEmpty}>
+                              {i18n.t('No tracked entity types available')}
```

Comment:
I believe most end users won't understand what "tracked entity types" refer to. How about using a message like this: "You don't have access to create" instead? Thanks!